inbm-vision/Changelog.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
  - HSD 15011298299, RTC 515263 - Support non-LTS versions of Ubuntu
+ - HSD 15011231779 - POTA rollback timer not following defined potaCompletionTimerSecs
```

Comment:
We don't want to put documentation changes in the changelog.   I know Abhishek told you all different when I was out, but we discussed and we don't want to do this.

Reply:
Updated changelog as per comments.